test/distributed/test_c10d_nccl.py

```diff
@@ -2958,6 +2958,23 @@ def test_collectives(self):
     def test_allreduce_coalesced(self):
         self._test_allreduce_coalesced(backend="nccl")
 
+    @requires_nccl()
+    @skip_if_lt_x_gpu(1)
+    def test_allgather_base(self):
+        store = dist.FileStore(self.file_name, self.world_size)
+        dist.init_process_group(
+            "nccl",
+            world_size=self.world_size,
+            rank=self.rank,
+            store=store,
+        )
+        device = "cuda"
+        tensor = torch.ones(10, 10, device=torch.device(device))
+        output_tensor = torch.zeros(10, 10, device=torch.device(device))
+        dist._all_gather_base(output_tensor, tensor)
```

Comment:
Please use `all_gather_into_tensor` at Python level.
We will also rename `_all_gather_base` at the binding and C++ levels to `all_gather_into_tensor` when we introduce extension-breaking changes in 2.0.

Reply:
good point! thanks